neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -23,18 +23,33 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation_fn: Callable[[tf.Tensor], tf.Tensor]=tf.nn.relu,
                  dropout_keep_prob: float=0.5,
                  save_checkpoint: Optional[str]=None,
```

Comment:
když uhryzáváš `Optional`, tak to dělej konzistentně, jinak to akorát zanáší bordel. Já osobně nevim, jak to přesně funguje, mám za to, že je to spíš zbytečný, když tomu dáváš None jako default hodnotu. Ale když budeš mít typ `List[str]=None`  a pak se v tý funkci změní default na něco jinýho a None zůstane přípustná hodnota, tak tam už to Optional bejt má, takže by asi bylo chytrý to tam nechávat všude.